assets/js/modules/ads/datastore/settings.test.js

```diff
@@ -109,6 +139,216 @@ describe( 'modules/ads settings', () => {
 				},
 			} );
 		} );
+
+		it( 'should handle an error when sending a POST request to the FPM settings endpoint', async () => {
```

Comment:
I added a couple of additional tests to extend the coverage for the existing behaviour.